### DISCUSSION TITLE
Option to have Last.fm scrobble first artist only

### DISCUSSION THREAD

**Discussion by ghost** [ciderapp/Cider#1600] [Feature Request]
Currently having LFM connected to Spotify and playing a collaboration will only send the first artist to LFM. For example _Stay_ by The Kid LAROI and Justin Bieber will show on LFM as by The Kid LAROI only. LFM sucks at handling collaborations so I think this is the best option available since at least one artist will have their scrobble count increased properly instead of creating a new The Kid LAROI & JB entry. It would be nice to have this option on Apple Music as well.

**Comment by coredev-uk** [comment]
turn on remove featured artists in the settings

**Reply by ghost** [reply]
Huh you're right, thanks. However I would suggest renaming that setting, because it says "Remove featuring artists from the song title" which sounds like it removes (feat. XYZ) when it's not affecting the song title at all.
